src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Connections.cs

```diff
@@ -43,6 +43,12 @@ internal static TcpConnectionInformation[] ParseActiveTcpConnectionsFromFiles(st
             string tcp6FileContents = File.ReadAllText(tcp6ConnectionsFile);
             string[] v6connections = tcp6FileContents.Split(s_newLineSeparator, StringSplitOptions.RemoveEmptyEntries);
 
+            // On WSL 1 the files are always empty.
+            if (v4connections.Length == 0 || v6connections.Length == 0)
+            {
+                throw ExceptionHelper.CreateForParseFailure();
+            }
```

Comment:
Is there ever a time where these being empty is not a parse failure? What happens if IPv6 is disabled and the user only has IPv4?

Reply:
On many Docker container scenarios, IPv6 isn't there. So, this might cause unexpected failures. So, perhaps the "||" should be an "&&".

I would suggest testing this PR change on a docker container. Most docker containers run on Docker for Windows don't have IPv6 networking enabled.

Reply:
I guess empty is always failure because the first line is expected to have a header, although I will test it on Docker.
Also there's a check on file existence just a few lines higher, which was already there. So if one file is missing it'll throw `PlatformNotSupportedException`.

Reply:
Just tested it on docker where only IPv4 is supported and the tests passed except for the ones for IPv6 loopback. The files are present, they contain the first line with header, just no records. So my change is not triggered even on docker with only IPv4 support.